### PR TITLE
feat: add table for proposal list page

### PR DESCRIPTION
**Pull request by bkioshn**
## Describe your changes
Add proposal list table to proposal list page

### CODE REVIEW

File: src/lib/pages/proposals/table/ProposalTableRow.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  const { currentChainName } = useWallet();
+
+  const getAddressType = useGetAddressType();
+  const columnsWidth = templateColumns?.toString().split(" ");
+  const isInactive = proposal.status === ProposalStatus.INACTIVE;
+  const isDepositOrVoting =
+    proposal.status === ProposalStatus.DEPOSIT_PERIOD ||
+    proposal.status === ProposalStatus.VOTING_PERIOD;
+
+  const hoverBg = () => {
```
Comment (poomthiti) [inline src/lib/pages/proposals/table/ProposalTableRow.tsx:36]: specify return type

**Review by poomthiti — CHANGES_REQUESTED** [review]

File: src/lib/pages/proposals/index.tsx [hunk 2]
```diff
@@ -1,20 +1,72 @@
...
+import { PorposalTable } from "./table/ProposalTable";
+
+const Proposals = () => {
+  const chainId = useChainId();
   const router = useRouter();
+  const { data: countProposals = 0 } = useProposalListCount(
+    [],
+    [],
+    "",
+    undefined
```
[the remark this answers is not in the document]
Comment (bkioshn) [reply src/lib/pages/proposals/index.tsx:27]: Will implement this later in another PR

File: src/lib/pages/proposals/table/ProposalTableRow.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+export const ProposalTableRow = ({
+  proposal,
+  templateColumns,
+  boxShadow,
+}: ProposalRowProps) => {
+  const { currentChainName } = useWallet();
+
+  const getAddressType = useGetAddressType();
+  const columnsWidth = templateColumns?.toString().split(" ");
+  const isInactive = proposal.status === ProposalStatus.INACTIVE;
```
Comment (evilpeach) [inline src/lib/pages/proposals/table/ProposalTableRow.tsx:31]: Let's use `isActive`

File: src/lib/pages/proposals/table/ProposalTableRow.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+export const ProposalTableRow = ({
+  proposal,
+  templateColumns,
+  boxShadow,
+}: ProposalRowProps) => {
+  const { currentChainName } = useWallet();
+
+  const getAddressType = useGetAddressType();
+  const columnsWidth = templateColumns?.toString().split(" ");
```
Comment (evilpeach) [inline src/lib/pages/proposals/table/ProposalTableRow.tsx:30]: add todo

File: src/lib/pages/proposals/table/ProposalTableRow.tsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  };
+
+  return (
+    <Grid
+      templateColumns={templateColumns}
+      minW="min-content"
+      cursor={!isInactive ? "pointer" : "default"}
+      _hover={{ "> div": { bgColor: hoverBg } }}
+      onClick={() =>
+        !isInactive ??
```
Comment (evilpeach) [inline src/lib/pages/proposals/table/ProposalTableRow.tsx:49]: isActive || ...

**Pull request merged**